### PR TITLE
feat: add mojo

### PR DESCRIPTION
**Pull request by ttytm**
The changes in the PR would add an icon for mojo.

- Used color is the linguist color
- Used icon is nf-md-fire as a font-icon looks better in the tree as a unicode emoji character and is less likely to set things off. If the emoji is the preferred choice, please let me know, I don't have a strong preference here.

### CODE REVIEW

File: lua/nvim-web-devicons/icons-default.lua [hunk 1]
```diff
@@ -3491,6 +3485,18 @@ local icons_by_operating_system = {
     cterm_color = "35",
     name = "Manjaro",
   },
+  ["mint"] = {
+    icon = "󰣭",
+    color = "#66af3d",
+    cterm_color = "70",
+    name = "Mint",
+  },
```
Comment (alex-courtis) [inline lua/nvim-web-devicons/icons-default.lua:3493]: Thank you.

Comment (alex-courtis) [inline lua/nvim-web-devicons/icons-default.lua:3493]: Thank you.

**Comment by alex-courtis** [comment]
> The changes in the PR would add an icon for mojo.
> 
> * Used color is the linguist color
> * Used icon is nf-md-fire as a font-icon looks better in the tree as a unicode emoji character and is less likely to set things off. If the emoji is the preferred choice, please let me know, I don't have a strong preference here.

Can I please ask you to use nf-fa-fire `""` instead?

We're preferring seti/fontawesome etc. over material design. A MD variant will come soon.

**Review by alex-courtis — APPROVED** [review]
Many thanks for your contribution!

File: lua/nvim-web-devicons/icons-default.lua [hunk 2]
```diff
@@ -3491,6 +3485,24 @@ local icons_by_operating_system = {
...
+  ["mojo"] = {
+    icon = "",
+    color = "#ff4c1f",
+    cterm_color = "196",
+    name = "Mojo",
+  },
+  ["🔥"] = {
```
Comment (alex-courtis) [inline lua/nvim-web-devicons/icons-default.lua:3500]: I'm not actually seeing this emoji during a hi test:


Which font does this emoji come from?

File: lua/nvim-web-devicons/icons-default.lua [hunk 3]
```diff
@@ -3491,6 +3485,24 @@ local icons_by_operating_system = {
     cterm_color = "35",
     name = "Manjaro",
   },
+  ["mint"] = {
+    icon = "󰣭",
+    color = "#66af3d",
+    cterm_color = "70",
+    name = "Mint",
+  },
+  ["mojo"] = {
```
Comment (alex-courtis) [inline lua/nvim-web-devicons/icons-default.lua:3494]: Apologies, I should have noticed earlier:

This is in the `icons_by_operating_system` wheras it should be in `icons_by_file_extension`.

You can run `:NvimWebDeviconsHiTest` to check.

**Review by alex-courtis — CHANGES_REQUESTED** [review]

File: lua/nvim-web-devicons/icons-default.lua (hunk 3 above)
Comment (ttytm) [reply lua/nvim-web-devicons/icons-default.lua]: Whoops. My fault. Just seen one big table and looked for the "m" section :see_no_evil:

File: lua/nvim-web-devicons/icons-default.lua (hunk 2 above)
Comment (ttytm) [reply lua/nvim-web-devicons/icons-default.lua]: https://docs.modular.com/mojo/faq default fire emoji

File: lua/nvim-web-devicons/icons-default.lua (hunk 3 above)
Comment (ttytm) [reply lua/nvim-web-devicons/icons-default.lua]: Thanks for the review work :100: !

File: lua/nvim-web-devicons/icons-default.lua (hunk 2 above)
Comment (alex-courtis) [reply lua/nvim-web-devicons/icons-default.lua]: OK. I installed a font `noto-font-emoji` and can see it in browser and terminal. We'll see how it goes.

I do not think an emoji as part of a file name is a good idea; most terminals won't support it and people need to install a font to be able to see it.

**Review by alex-courtis — APPROVED** [review]

**Comment by alex-courtis** [comment]
Many thanks for your contribution!

**Pull request merged**